src/SFA.DAS.QnA.Api/Startup.cs

```diff
@@ -93,8 +94,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IApplicationDataValidator, ApplicationDataValidator>();
             
             services.AddTransient<IEncryptionService, EncryptionService>();
+            services.AddTransient<INotRequiredProcessor, NotRequiredProcessor>();
```

Comment:
Is Transient correct? Could it be made into a Singleton?

Reply:
This is likely to be a mute point as all of the others are Transient 🤷‍♂

Reply:
Still seems like a singleton to me tho 👍